src/components/App/index.js

```diff
@@ -103,9 +104,20 @@ class ModalApp extends Component {
         this.onInvalidJWT('hideOnfidoLogo feature not enabled for this account.')
       }
 
-    } else if (!options.mobileFlow && token && token !== prevToken) {
+    } else if (!options.mobileFlow && isNewToken) {
       this.props.actions.hideOnfidoLogo(false)
     }
+    if (options.enterpriseFeatures?.cobrand && !isHideOnfidoLogoEnabled && isNewToken) {
+      const validEnterpriseFeatures = getEnterpriseFeaturesFromJWT(token)
```

Comment:
Should we store these values in the redux store so we don't have to call this function to parse the token every time? Or you can just call it at the beginning of the function and delete it from line 97 too.

Reply:
I think it's only used here, so we could call it at the beginning of the function to start with.